ansible/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Docker YUM depencies present
+  tags:
+    - docker
+  yum:
+    name: ['yum-utils', 'device-mapper-persistent-data', 'lvm2']
+    state: present
+
+- name: Docker YUM repo present
+  tags:
+   - docker
+  yum_repository:
+    name: docker-ce
+    description: Docker-ce YUM repo
+    baseurl: https://download.docker.com/linux/centos/7/x86_64/stable
```

Comment:
Add following line:
gpgkey: https://download.docker.com/linux/centos/gpg

Following step currently fails from fresh install with error: TASK [docker : Docker present] ***************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "changes": {"installed": ["docker-ce"]}, "msg": "warning: /var/cache/yum/x86_64/7/docker-ce/packages/docker-ce-18.09.7-3.el7.x86_64.rpm: Header V4 RSA/SHA512 Signature, key ID 621e9f35: NOKEY\n\n\nPublic key for docker-ce-18.09.7-3.el7.x86_64.rpm is not installed\n"

Reply:
Done